pkg/dockerutil/dockerutils.go

```diff
@@ -550,7 +560,7 @@ func ComposeCmd(composeFiles []string, action ...string) (string, string, error)
 	// Container (or Volume) ... Creating or Created or Stopping or Starting or Removing
 	// Container Stopped or Created
 	// No resource found to remove (when doing a stop and no project exists)
-	ignoreRegex := "(^ *(Network|Container|Volume) .* (Creat|Start|Stopp|Remov)ing$|^Container .*(Stopp|Creat)(ed|ing)$|Warning: No resource found to remove$|Pulling fs layer|Waiting|Downloading|Extracting|Verifying Checksum|Download complete|Pull complete)"
+	ignoreRegex := "(^ *(Network|Container|Volume) .* (Creat|Start|Stopp|Remov)ing$|^Container .*(Stopp|Creat)(ed|ing)$|Warning: No resource found to remove|Pulling fs layer|Waiting|Downloading|Extracting|Verifying Checksum|Download complete|Pull complete)"
```

Comment:
I removed `$` from `Warning: No resource found to remove$` because there was extra output from `docker-compose` with this command:

```
$ project=test-5305 && mkdir $project && cd $project && ddev config --auto --omit-containers=db && ddev start && ddev stop && ddev restart && ddev delete --omit-snapshot --yes && cd .. && rm -rf $project
...
Starting test-5305...
...
Project test-5305 has been stopped. 
Restarting project test-5305... 
Warning: No resource found to remove for project "ddev-test-5305".
...
```